rewrite-java/src/main/java/org/openrewrite/java/dataflow/internal/csv/CsvLoader.java

```diff
@@ -51,4 +51,32 @@ private static <R extends Mergeable<R>, E> R loadCvs(InputStream input, URI sour
         }
     }
 
+    private static String[] parseLine(String line) {
+        if (line.isEmpty()) {
+            return new String[0];
+        }
+
+        ArrayList<String> result = new ArrayList<>();
```

Comment:
You could theoretically pre-size this as a max size of 10

Reply:
Which happens to be the default capacity of an `ArrayList` 😀. If we wanted to optimize this, there would be other means too, such as always reusing the same array (of the correct length). But it's a meager 10K rows, so I didn't bother.

Reply:
> Which happens to be the default capacity of an ArrayList.

Oh, wow, didn't know this. Thanks! 😆